third-party/cmake/Libvterm-tbl2inc_c.cmake

```diff
@@ -0,0 +1,163 @@
+cmake_minimum_required(VERSION 2.8.7)
+
+set(HEX_ALPHABET "0123456789abcdef")
+
+function(ConvertToHex dec hex)
+  while(dec GREATER 0)
+    math(EXPR _val "${dec} % 16")
+    math(EXPR dec "${dec} / 16")
+    string(SUBSTRING ${HEX_ALPHABET} ${_val} 1 _val)
+    set(_res "${_val}${_res}")
+  endwhile()
+  # Pad the result with the number of zeros
+  # specified by the optional third argument
+  if(${ARGC} EQUAL 3)
+    set(padding ${ARGV2})
+    string(LENGTH ${_res} _resLen)
+    if(_resLen LESS ${padding})
+      math(EXPR _neededPadding "${padding} - ${_resLen}")
+      foreach(i RANGE 1 ${_neededPadding})
+	set(_res "0${_res}")
```

Comment:
I see this unindented for some reason.

Reply:
Sorry, I accidentally inserted tabs in some places because I wasn't using `expandtab`. It's fixed now.